docs/configuration/files.md

```diff
@@ -5,6 +5,12 @@ uv supports persistent configuration files at both the project- and user-level.
 Specifically, uv will search for a `pyproject.toml` or `uv.toml` file in the current directory, or
 in the nearest parent directory.
 
+!!! note
+
+    For user-level commands, like `uv tool install` or `uv python install`, local configuration
```

Comment:
Minor problem here, `uv python install` without arguments reads from `.python-version` (and in the future `pyproject.toml`), it might need to respect project settings.

Reply:
Ok, I can back that part out?

Reply:
Technically it should probably ignore project settings unless implicitly installing versions for that project, but that seems complicated to explain... I guess I'd back it out entirely for now?